packages/component/src/Utils/FocusRedirector.tsx

```diff
@@ -19,6 +19,11 @@ type FocusRedirectorProps = {
 };
 
 const FocusRedirector: FC<FocusRedirectorProps> = ({ className, onFocus, redirectRef }) => {
+  // Hide from screen readers when not needed
+  const ariaHidden = !redirectRef?.current ? 'true' : 'false';
+  const TAB_INDEX_DISABLED = -1;
+  const TAB_INDEX_ENABLED = 0;
+
```

Comment:
```suggestion
  // Hide from screen readers when not in use to prevent unnecessary announcements
  const shouldHide = !!redirectRef?.current;

```

Reply:
fixed